std-bits/tableau/src/main/java/org/enso/tableau/HyperFormat.java

```diff
@@ -355,6 +357,11 @@ private static TableDefinition createTable(String schemaName, String tableName,
           case DateType _ -> tableDef.addColumn(columnName, SqlType.date());
           case TimeOfDayType _ -> tableDef.addColumn(columnName, SqlType.time());
           case DateTimeType _ -> tableDef.addColumn(columnName, SqlType.timestampTz());
+          // https://tableau.github.io/hyper-db/docs/sql/datatype/numeric
+          // Precisions over 18 require 128-bit for internal storage. Processing 128-bit numeric values is 
+          // often slower than processing 64-bit values, so it is advisable to use a sensible precision for 
+          // the use case at hand instead of always using the maximum precision by default.
+          case BigDecimalType _ -> tableDef.addColumn(columnName, SqlType.numeric(18, 9));
```

Comment:
Do we have a ticket to revisit this - so we can preserve precision and scale?

Reply:
https://github.com/enso-org/enso/issues/13022